.coveragerc

```diff
@@ -4,6 +4,10 @@ source_pkgs = incremental
 # List of directory names.
 source = tests
 branch = True
+parallel = True
+# This must be an absolute path because the example tests
+# run Python processes with alternative working directories.
+data_file = ${TOX_INI_DIR-.}/.coverage
```

Comment:
NB: `coverage` also supports `pyproject.toml` now too https://coverage.readthedocs.io/en/7.5.4/config.html#sample-file (although let's migrate to that in a different PR)

Reply:
Yeah I figured there's waaaay too much random cleanup here already. (I'd also like to migrate to Ruff, etc.)